src/EspMQTTClient.h

```diff
@@ -29,6 +29,17 @@
 
 #endif
 
+class EspMQTTClient; // forward declaration
```

Comment:
Is this needed considering that the class definition is 4 rows below?

Reply:
Thanks, probably I just left it there from before. Works fine without it.